packages/client/src/v2-events/features/events/actions/print-certificate/pdfUtils.ts

```diff
@@ -304,11 +330,16 @@ export function compileSvg({
    */
   function $lookup(obj: EventMetadata | EventState, propertyPath: string) {
     const stringifyDeclaration = getFormDataStringifier(intl, locations)
+    const fieldConfigs = config.declaration.pages.flatMap((x) => x.fields)
 
-    const resolvedDeclaration = stringifyDeclaration(
-      config.declaration.pages.flatMap((x) => x.fields),
-      $declaration
-    )
+    // Sometimes we want to access certain fields as specific object keys instead of plain strings.
+    // For example, we want to access birthLocation as an object with keys FACILITY, DISTRICT, STATE, COUNTRY.
```

Comment:
@cibelius This case should work similarly as with `Address` component meaning `getFormDataStringifier` should produce, for birthLocation (type `FACILITY`) an object type with all the keys you need. `LocationSearch` might already support this in its `Output` but if you need something different for FACILITY specifically, I'd recommend you make a new registered component that internally uses `LocationSearch` but has a different Output

Reply:
Oh, it seems LocationSearch doesn't even have a `stringify` yet. Maybe this is a good time to implement it